testing/gnome-menus/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
```

Comment:
You can add yourself as contributor if you want credit for your work :)

Reply:
I didn't really create the APKBUILD, it's coming from the pmaports tree from postmarketOS, so it's fine :)